Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -796,59 +835,47 @@ driverInfoCardViewState state = { props:
       , vehicleNo = HU.makeNumber $ state.data.driverInfoCardState.registrationNumber     
-      , suggestionsList = if showSuggestions state then if (metersToKm state.data.driverInfoCardState.distance state) == getString AT_PICKUP then getSuggestionsfromKey "customerInitialAP" else getSuggestionsfromKey "customerInitialBP" else state.data.suggestionsList
+      , suggestionsList = getChatSuggestions state
       , hint = (getString MESSAGE)
-      , suggestionHeader = (getString START_YOUR_CHAT_USING_THESE_QUICK_CHAT_SUGGESTIONS)
-      , emptyChatHeader = (getString START_YOUR_CHAT_WITH_THE_DRIVER)
       , languageKey = (getValueToLocalStore LANGUAGE_KEY)
-      , mapsText = "Maps"
-      , grey700 = Color.grey700
-      , blue600 = Color.blue600
-      , blue900 = Color.blue900
-      , transparentGrey = Color.transparentGrey
-      , green200 = Color.green200
-      , grey900 = Color.grey900
-      , grey800 = Color.grey800
-      , blue800 = Color.blue800
-      , white900 = Color.white900
-      , black800 = Color.black800
-      , black700 = Color.black700
       , canSendSuggestion = state.props.canSendSuggestion
-      , showAutoGeneratedText = (getValueToLocalStore NOTIFIED_CUSTOMER == "true") && (HU.secondsToHms state.data.driverInfoCardState.eta) /= ""
+      , showAutoGeneratedText = (getValueToLocalStore NOTIFIED_CUSTOMER == "true") && isJust state.data.driverInfoCardState.eta && (HU.secondsToHms $ fromMaybe 0 state.data.driverInfoCardState.eta) /= "--"
       , rideConfirmedAt = state.data.driverInfoCardState.startedAt
-      , autoGeneratedText = state.data.config.notifyRideConfirmationConfig.autoGeneratedText <> (HU.secondsToHms state.data.driverInfoCardState.eta)
+      , autoGeneratedText = state.data.config.notifyRideConfirmationConfig.autoGeneratedText <> (HU.secondsToHms $ fromMaybe 0 state.data.driverInfoCardState.eta)
+      , driverRating = show $ state.data.driverInfoCardState.rating
+      , fareAmount = show $ state.data.driverInfoCardState.price
+      , config = state.data.config
+      , peekHeight = if state.data.infoCardPeekHeight == 0 then ((if state.props.currentStage == RideAccepted then 345 else 328) + if true then 25 else 0) else state.data.infoCardPeekHeight
+      , otp = state.data.driverInfoCardState.otp
       , enableSuggestions = state.data.config.feature.enableSuggestions
   }
-  in chatViewConfig'
+  in messagingViewConfig'
 
-showSuggestions :: ST.HomeScreenState -> Boolean
-showSuggestions state = (not $ state.data.lastMessage.sentBy == "Customer") && DA.null state.data.suggestionsList && ((show $ DA.length $ JB.getChatMessages "") == state.data.messagesSize || state.data.messagesSize == "-1")
+getDefaultPeekHeight :: ST.HomeScreenState -> Int
+getDefaultPeekHeight state = if state.data.currentSearchResultType == ST.QUOTES then (if state.props.currentStage == ST.RideAccepted then 234 else 334) else ((if state.props.currentStage == ST.RideAccepted then 339 else 318))
```

Comment:
@PraveenGongada use a base value and add additional height on top of it and use a function to retrieve additional Height.

Reply:
Don't have any base height here, It's all single component, the height comes from layout bounds function, initially it will be 0, this was added to handle that case